src/Entity/Cover.php

```diff
@@ -88,7 +77,16 @@ class Cover
     /**
      * @var string|null
      *
-     * @ApiProperty(iri="http://schema.org/contentUrl")
+     * @ApiProperty(
+     *     iri="http://schema.org/contentUrl",
+     *     attributes={
+     *         "openapi_context"={
+     *             "type"="string",
+     *             "format"="url",
+     *             "example"="https://upload.cover.dandigbib.org/cover/5ed65baa2b264_870970-basis%3A52890365.jpg"
```

Comment:
Not sure that I understand this example? Is this the output when the cover have been uploaded?

Reply:
It enables this output:
<img width="1071" alt="Screenshot 2020-06-03 at 10 45 36" src="https://user-images.githubusercontent.com/5631988/83615905-72001f00-a587-11ea-81a8-b1aa4d967624.png">

Current output for reference:
<img width="735" alt="Screenshot 2020-06-03 at 10 47 47" src="https://user-images.githubusercontent.com/5631988/83616116-bb506e80-a587-11ea-9510-b22f304e75fb.png">